Sources/UIKit/UIColor.swift

```diff
@@ -37,6 +37,27 @@ public extension UIColor {
         self.init(red: components.red, green: components.green, blue: components.blue, alpha: components.alpha)
     }
 
+    var hexString: String {
+
+        var components = UIColor.components(fromHex6: 0)
+        getRed(&components.red, green: &components.green, blue: &components.blue, alpha: &components.alpha)
+        let rgb: Int = (Int)(components.red*255)<<16
+            | (Int)(components.green*255)<<8
+            | (Int)(components.blue*255)<<0
+        return String(format:"#%06x", rgb)
+    }
+
+    var hexStringWithAlpha: String {
+
+        var components = UIColor.components(fromHex6: 0)
+        getRed(&components.red, green: &components.green, blue: &components.blue, alpha: &components.alpha)
+        let argb: Int = (Int)(components.alpha * 255)<<24
```

Comment:
I'm a bit confused right now. It seems we are using `#RRGGBBAA` when **encoding** a color to hex, but using `#AARRGGBB` when **decoding** from hex 🤔 

Still, I can't find any type of color standard for hex representation with alpha channel...

We should definitely set this straight and settle on one format, and also properly document the API's, imo.

cc  @portellaa